weatherbot/main.py

```diff
@@ -9,49 +9,19 @@
 
 import requests
 import json
-from weatherbot.get_temp import get_weather
-from weatherbot.tokens import TELEGRAM_TOKEN
-
-URL = "https://api.telegram.org/bot" + TELEGRAM_TOKEN + "/"
-
-def get_updates():
-    # получаю обновления от сервера телеграм в виде словаря в формате json :
-    url = URL + "getUpdates"
-    response = requests.get(url)
-    return response.json()
-
-def get_message():
-    # в словаре ищу id пользователя и его text, записываю их в переменные
-    # и возвращаю себе словарь message, состоящий только из этих переменных, [-1] - чтобы взять последний элемент
-    data = get_updates()
-    chat_id = data["result"][-1]["message"]["chat"]["id"]
-    message_text = data["result"][-1]["message"]["text"]
-
-    message = {"chat_id": chat_id,
-                "text": message_text}
-    return message
-
-def send_message(chat_id, text):
-    url = URL + "sendMessage?chat_id={}&text={}".format(chat_id, text)
-    requests.get(url)
+from weatherbot.get_temp import get_weather, conv_kelv_to_cels, add_C
+from weatherbot.tokens import OWN_TOKEN
 
 def main():
-    # dictionary = get_updates()
-
-    # with open("updates.json", "w", encoding='utf-8') as jsonfile:
-    #      json.dump(dictionary, jsonfile, indent=2, ensure_ascii=False)
-    # while True:
-        answer = get_message()
-        chat_id = answer["chat_id"]
-        text = answer["text"]
-
-        if '/start' in text:
-            send_message(chat_id, "Hi, I can help you to know the weather, just tell me the city.")
-        elif '/city' in text:
-            send_message(chat_id, get_weather())
-        else:
-            send_message(chat_id, "If you want to know the weather just send me a city.")
-
+    if len(sys.argv) == 2:
+        program_name = sys.argv[0]
+        city = sys.argv[1]
+        weather = get_weather(city, OWN_TOKEN)
+        temp = conv_kelv_to_cels(weather)
+        cels = add_C(temp)
+        print(cels)
+    else:
+        print("Please, enter the city")
```

Comment:
What happens if I provide more? E.g. ` python3 -m weatherbot.main Krasnoyarsk Tomsk`? It tells me `Please, enter the city`, but I provided the city - I think error message can be more specific.

Reply:
Thx! I will fix it.